userdetails-cognito/src/main/groovy/au/org/ala/userdetails/CognitoUserService.groovy

```diff
@@ -269,17 +282,9 @@ class CognitoUserService implements IUserService {
                         }
                     }
 
-            UserRecord user = new UserRecord(
-                    dateCreated: userResponse.user.userCreateDate,
-                    lastUpdated: userResponse.user.userLastModifiedDate,
-                    activated: userResponse.user.userStatus == "CONFIRMED", locked: !userResponse.user.enabled,
-                    firstName: attributes.find { it.key == 'given_name' }.value,
-                    lastName: attributes.find { it.key == 'family_name' }.value,
-                    email: attributes.find { it.key == 'email' }.value,
-                    userName: userResponse.user.username,
-                    userProperties: userProperties
-            )
+            UserRecord user = cognitoUserTypeToUserRecord(userResponse.user, true)
 
+            //tODO need to change this
```

Comment:
Should we address this TODO before merging?

Reply:
Fixed